i18n/en.json

```diff
@@ -1857,13 +1861,16 @@
   "upload_success": "Upload success, refresh the page to see new upload assets.",
   "upload_to_immich": "Upload to Immich ({count})",
   "uploading": "Uploading",
+  "id": "ID",
   "url": "URL",
   "usage": "Usage",
   "use_current_connection": "use current connection",
   "use_custom_date_range": "Use custom date range instead",
   "user": "User",
   "user_id": "User ID",
   "user_liked": "{user} liked {type, select, photo {this photo} video {this video} asset {this asset} other {it}}",
+  "created_at": "Created",
+  "updated_at": "Updated",
```

Comment:
The translation does not really match the key, does it?

Reply:
I thought `created` by itself was ambiguous. Like  "created" (success!) vs "created" (label for a date), etc.